nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -425,15 +434,15 @@ contract StakingEscrow is Issuer {
         StakerInfo storage info = stakerInfo[msg.sender];
         require(_worker != info.worker, "Specified worker is already set for this staker");
         uint16 currentPeriod = getCurrentPeriod();
-        if(info.worker != address(0)){ // If this staker had a worker ...
+        if (info.worker != address(0)) { // If this staker had a worker ...
             // Check that enough time has passed to change it
             require(currentPeriod >= info.workerStartPeriod.add16(minWorkerPeriods),
                 "Not enough time has passed since the previous setting worker");
             // Remove the old relation "worker->staker"
             workerToStaker[info.worker] = address(0);
         }
 
-        if (_worker != address(0)){
+        if (_worker != address(0)) {
             require(workerToStaker[_worker] == address(0), "Specified worker is already in use");
             require(stakerInfo[_worker].subStakes.length == 0 || _worker == msg.sender,
                 "Specified worker is a staker");
```

Comment:
Yep, I removed from one-time method on purpose, 
but I hope this require messages will be useful or it will be just a dead weight for next optimization time